src/G2/Config/Config.hs

```diff
@@ -77,6 +83,7 @@ mkConfig homedir = Config Regular
     <*> pure []
     <*> mkLogMode
     <*> flag Sharing NoSharing (long "no-sharing" <> help "disable sharing")
+    <*> flag InstBefore InstAfter (long "inst-after" <> help "instantiate type variables in the function")
```

Comment:
Update help text- we're instantiate either way, need to clarify instantiating before or instantiating after.

Reply:
I am a bit confused, didn't I clarify the inst-after from the text, so did you want me to add the clarification for the default option: inst before in the text also?